hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java

```diff
@@ -716,7 +716,7 @@ private HapiSpec wipeRespectsConstraints() {
                         cryptoTransfer(movingUnique(NFT, 1, 2).between(TOKEN_TREASURY, ACCOUNT)))
                 .when()
                 .then(wipeTokenAccount(
-                                NFT, ACCOUNT, LongStream.range(0, 1000).boxed().collect(Collectors.toList()))
+                                NFT, ACCOUNT, LongStream.range(1, 1001).boxed().collect(Collectors.toList()))
```

Comment:
will be nice to add a comment why we changed this, as discussed

Reply:
Yep sure 👍 
This was changed because the test itself is (slightly) invalid: NFT ID's have to be >= 1. This test was failing with `INVALID_NFT_ID` in the modular code because we moved the batch check to `handle()`, whereas the mono code checked the batch size in prehandle, and thus failed before it could check the validity of the NFT ID's.